ACC2024/RobotDynamics/@RigidBody2D/RigidBody2D.m

```diff
@@ -46,9 +46,15 @@
             end
 
         end
-
+        
+        % [x,y,theta]
         function position = forwardKinematics(obj, pointInBodyFrame)
             position = obj.CoordinateFrame.HomogeneousFromBase * [pointInBodyFrame.ColumnVector; 1];
+            position = [position; obj.CoordinateFrame.RotationFromBase.Theta];
```

Comment:
Where is this function used? You need to change the usage.

Reply:
OK. Since we decide to use circular foot, we don't need this function any more